cockatrice/src/window_main.cpp

```diff
@@ -932,9 +932,9 @@ void MainWindow::createTrayIcon()
 void MainWindow::iconActivated(QSystemTrayIcon::ActivationReason reason)
 {
     if (reason == QSystemTrayIcon::DoubleClick) {
-        if (windowState() != Qt::WindowMinimized && windowState() != Qt::WindowMinimized + Qt::WindowMaximized)
+        if ((windowState() & (Qt::WindowMinimized & Qt::WindowMaximized)) == 0) {
```

Comment:
um, I'm not sure what you're doing here but doing & on these flags will always be 0

Reply:
```suggestion
        if (windowState() & Qt::WindowMinimized == 0) {
```

Reply:
Ahh I meant to do `|`

Reply:
```suggestion
        if ((windowState() & (Qt::WindowMinimized | Qt::WindowMaximized)) == 0) {
```

Reply:
```suggestion
        if ((windowState() & Qt::WindowMinimized) == 0) {
```

Paren required for bitwise